### DISCUSSION TITLE
Incendiary explosions weak and underwhelming

### DISCUSSION THREAD

**Discussion by irRegularGuy646** [CombatExtended-Continued/CombatExtended#2369] [General]
the T-9's fuel cells just start like 8 fires out of some 100 odd tiles in its blast radius, even 105mm artillery shells that each apparently contain about 1.75 kg of promethium each never do more than just minor promethium coating and some sparse fires. In general, most incendiary explosions are just underwhelming and weak.
![image](https://user-images.githubusercontent.com/71989367/222333580-8a10dd1b-12d8-42c7-a9ef-1c6b0e1a7804.png)
Fig 1: the effect of 5.25 kgs of promethium dispersed via 105mm artillery bombardment (3 shells), immediately after impact

![image](https://user-images.githubusercontent.com/71989367/222334121-57d16b86-fae2-4237-86dc-804aa9c289cb.png)
what i'd prefer for it to be like

**Comment by irRegularGuy646** [comment]
i just did some testing with the T-9, even direct hits with incendiary fuel cells failed to apply even minor promethium soaking, or even any damage at all.

**Comment by SamaelGray** [comment]
This fixes the issue:

https://github.com/CombatExtended-Continued/CombatExtended/pull/2373
